[#62295] Update Internal comments "Learn more" link to point to correct documentation (#18828)
opf/openproject: docs/user-guide/activity/README.md

- You can edit the comment if you are the author  
- You can quote someone else's comment in a new comment. This is useful if you want to respond to a certain part of a comment.

### Internal comments (Enterprise add-on)

> [!NOTE]
> Internal comments are an Enterprise add-on and can only be used with [Enterprise cloud](../../enterprise-guide/enterprise-cloud-guide/) or [Enterprise on-premises](../../enterprise-guide/enterprise-cloud-guide/). An upgrade from the free Community edition is easily possible.

> You can add work packages with restricited visibility to better manage information flow and share sensitive information on a need to know basis. Comments with restricted visibility are called **internal comments** in OpenProject. You will recognize internal comments by distinctinctly different color scheme and the lock icon. 

Reviewer: ```suggestion
You can add work package comments as internal to better manage information flow and share sensitive information to an authorized group of users. Internal comments are distinguished from other comments via a different color scheme and a lock icon. 
```

Reviewer: AFAIK, the term "restricted visibility" was completely replaced with "Internal comments" as the more clear option

![Example of an internal comment displayed under Activity tab of an OpenProject work package](openproject_user_guide_internal_comment_example.png)

> [!TIP]
> In order for internal comments feature to be used, correct permissions need to be granted to specific roles. We recommend creating a dedicated role. When a permission is removed, internal comments will no longer be visible. 
> 

> > In addition, internal comments need to be activated under [project settings](../projects/project-settings/work-packages/#work-package-internal-comments-enterprise-add-on/). 

Author: ```suggestion
> In addition, internal comments need to be activated under [project settings](../projects/project-settings/work-packages/#work-package-internal-comments-enterprise-add-on). 
```
